### PR TITLE
Calculated web requestor passivation timeout based on alb_stickiness_…

### PR DESCRIPTION
**Pull request by nowam1**
Calculated web requestor passivation timeout based on alb_stickiness_lb_cookie_duration_seconds and requestorPassivationToAlbStickinessMargin.
Configured web requestor passivation timeout in prconfig based on calculated value.

### CODE REVIEW

**Comment by dcasavant** [comment]
Is any documentation change in the readme or community required for this? 

@taz-mon

**Comment by nowam1** [comment]
> Is any documentation change in the readme or community required for this?
> 
> @taz-mon

Probably description of new option: requestorPassivationToAlbStickinessMargin is required

File: charts/pega/templates/_helpers.tpl [hunk 1]
```diff
@@ -187,3 +187,14 @@ readinessProbe:
     Background
   {{- end -}}
 {{- end }}
+
+# Load balancer session cookie stickiness time in seconds,
+# calculated as sum of .requestor.passivationTimeSec and passivation delay.
+{{- define "lbSessionCookieStickiness" }}
```
Comment (dcasavant) [inline charts/pega/templates/_helpers.tpl]: Is it possible to add a check after the to see if `alb_stickiness_lb_cookie_duration_seconds` is set and use that value if the new configuration is not present?  That will preserve backwards compatibility and not require client changes of their values.yaml files.

File: charts/pega/config/deploy/context.xml.tmpl [hunk 2]
```diff
@@ -28,4 +28,7 @@
   <Environment name="prconfig/database/databases/CustomerData/defaultSchema" value="{{ .Env.DATA_SCHEMA }}" type="java.lang.String" />
   {{ end }}
   <Environment name="prconfig/database/drivers" value="com.microsoft.sqlserver.jdbc.SQLServerDriver;oracle.jdbc.OracleDriver;org.postgresql.Driver;com.ibm.db2.jcc.DB2Driver" type="java.lang.String" />
+  {{ if .Env.WEB_REQUESTOR_PASSIVATION_TIMEOUT }}
+  <Environment name="prconfig/timeout/browser" value="{{ .Env.WEB_REQUESTOR_PASSIVATION_TIMEOUT }}" type="java.lang.String" />
```
Comment (dcasavant) [inline charts/pega/config/deploy/context.xml.tmpl]: Remove WEB_ and just call the variable REQUESTOR_PASSIVATION_TIMEOUT to match the Helm parameter?

Comment (nowam1) [reply charts/pega/config/deploy/context.xml.tmpl]: Changed the name to REQUESTOR_PASSIVATION_TIMEOUT

File: charts/pega/templates/_helpers.tpl (hunk 1 above)
Comment (nowam1) [reply charts/pega/templates/_helpers.tpl]: Good idea. Added a check for alb_stickiness_lb_cookie_duration_seconds after checking that new configuration values are missing. If both new and old configs are present new will take precedence. Added one more check for presence of `requestor` block as it is missing in old configs and anyway it is optional.

**Pull request merged**